Formula/vcpkg.rb

```diff
@@ -0,0 +1,29 @@
+class Vcpkg < Formula
+  desc "C++ Library Manager for Windows, Linux, and MacOS"
+  homepage "https://docs.microsoft.com/en-us/cpp/vcpkg"
+
+  stable do
+    url "https://github.com/microsoft/vcpkg/archive/2019.09.tar.gz"
+    sha256 "16c1b151cc3cf2022b03b699970e548f48cd9cd1e68db0b9c23a1d0d39391e56"
+  end
+
+  head do
+    url "https://github.com/microsoft/vcpkg.git"
+  end
```

Comment:
```suggestion
  url "https://github.com/microsoft/vcpkg/archive/2019.09.tar.gz"
  sha256 "16c1b151cc3cf2022b03b699970e548f48cd9cd1e68db0b9c23a1d0d39391e56"
  head "https://github.com/microsoft/vcpkg.git"
```

This whole block can be replaced